.eslintrc

```diff
@@ -90,10 +90,10 @@
     "no-use-before-define": 2,
     "radix": 2,
     "curly": 2,
-    "no-fallthrough": 0,
-    "default-case": 0,
-    "no-plusplus": 0,
-    "dot-notation": 0
+    "no-fallthrough": 2,
+    "default-case": 2,
+    "no-plusplus": 2,
```

Comment:
Loop speed isn't ever going to be a bottleneck, always use HOFs when possible.

Want a for loop?
_.each(_.range(0, 4), function (i) {
});
Even then, you should avoid using Each because of the implication of side effects.